semantics/language/common/expr/bitwise.k

```diff
@@ -111,14 +111,14 @@ module C-COMMON-EXPR-BITWISE
           [structural]
      rule (.K => UNDEF("CEB6",
                "the right operand in a bitwise shift is negative.",
-               "6.5.7:3"))
+               "6.5.7:3, J.2:1 item 51"))
```

Comment:
Fixed. Thanks for pointing out. The format I've used, for the annex is, J.2:1 item X, and every section is comma separated.